### PR TITLE
Move security services

### PR DESCRIPTION
**Pull request by forestik**

### CODE REVIEW

**Review by Yura09 — APPROVED** [review]

File: service-api/pom.xml [hunk 1]
```diff
@@ -48,5 +48,19 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-context</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt</artifactId>
+            <version>0.9.1</version>
```
Comment (vterlyha) [inline service-api/pom.xml:54]: Please, move version to properties section

File: service/pom.xml [hunk 2]
```diff
@@ -99,6 +99,16 @@
             <artifactId>commons-fileupload</artifactId>
             <version>${commons-fileupload.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.social</groupId>
+            <artifactId>spring-social-facebook</artifactId>
+            <version>2.0.3.RELEASE</version>
```
Comment (vterlyha) [inline service/pom.xml:105]: Same here

File: service/src/test/java/greencity/security/service/OwnSecurityServiceImplTest.java [hunk 3]
```diff
@@ -121,18 +120,19 @@ public void init() {
...
     @Test
     void signUp() {
-        User user = User.builder().verifyEmail(new VerifyEmail()).build();
-        UserVO userVO = UserVO.builder().verifyEmail(new VerifyEmailVO()).build();
+        User user = User.builder()
+                .id(1L)
+                .verifyEmail(new VerifyEmail()).build();
+        UserVO userVO = UserVO.builder()
+                .id(1L)
+                .verifyEmail(new VerifyEmailVO()).build();
```
Comment (vterlyha) [inline service/src/test/java/greencity/security/service/OwnSecurityServiceImplTest.java:131]: Please, move this to ModelUtils class. It is also better to use Random class to generate numeric values

**Review by vterlyha — APPROVED** [review]
Minor fixes should be made

**Review by Markiyan000 — APPROVED** [review]

**Review by KhonkoDmytro — APPROVED** [review]

File: service/pom.xml (hunk 2 above)
Comment (forestik) [reply service/pom.xml]: Done

File: service/src/test/java/greencity/security/service/OwnSecurityServiceImplTest.java (hunk 3 above)
Comment (forestik) [reply service/src/test/java/greencity/security/service/OwnSecurityServiceImplTest.java]: Done

File: service-api/pom.xml (hunk 1 above)
Comment (forestik) [reply service-api/pom.xml]: Done

**Pull request merged**